tools/bioimaging/bioimage_inference.xml

```diff
@@ -0,0 +1,79 @@
+<tool id="bioimage_inference" name="Process image using a BioImage.IO model" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>with PyTorch</description>
+    <macros>
+        <token name="@TOOL_VERSION@">2.3.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <creator>
+        <organization name="BioImage.IO" url="https://bioimage.io/#/" email="" />
+        <person name="Beatriz Serrano-Solano" email="" />
+        <person name="Leonid Kostrykin" email="leonid.kostrykin@bioquant.uni-heidelberg.de" />
```

Comment:
@anuprulez you forgot yourself here.

@beatrizserrano your email?

I would probably use `<person givenName="" familyName="" email="" />`